complianceio/catalog.py

```diff
@@ -52,21 +54,21 @@ def create_groups(p):
             group = control_id[:2].lower()
             name = row[c_map.get('name')].value
             control_text = row[c_map.get('control')].value
-            implementation = row[c_map.get('implementation')].value
-            responsiblity = row[c_map.get('responsiblity')].value
+            responsibility = row[c_map.get('responsiblity')].value
             related = row[c_map.get('responsiblity')].value
             reference = row[c_map.get('reference')].value
             baseline = row[c_map.get('baseline')].value
+            priority = row[c_map.get('priority')].value
 
-            # ignore duplicated parts (may be better to use these instead)
+            # ignore duplicated parts (elements)
             if baseline is None:
                 continue
-            # check for ac--09(02)_smt
+            # check for ac--09(02)
             control_id = control_id.replace('--', '-')
             # check for AU14-(01)
             control_id = re.sub(r'([A-Z][A-Z])([0-9][0-9])\-', r"\1-\2", control_id)
-            # replace S1- with SI-
-            control_id = re.sub(r'S1-([0-9][0-9])', r"SI-\1", control_id)
+#            # replace S1- with SI-
+#            control_id = re.sub(r'S1-([0-9][0-9])', r"SI-\1", control_id)
 
```

Comment:
Is the only change here the leading space.

Reply:
Lines 70-71 were commented out as the spreadsheet no longer mislabels SI (eye) as S1 (one)